mis_builder/models/mis_report.py

```diff
@@ -354,7 +361,7 @@ def name_search(self, name="", args=None, operator="ilike", limit=100):
             ]
         )
         domain = osv_expression.AND([domain, name_search_domain])
-        return self.search(domain, limit=limit).name_get()
+        return self.search(domain, limit=limit)._compute_display_name()
```

Comment:
```suggestion
        return self._search(domain, limit=limit, order=order)
```

This is what `_name_search` must return.

Reply:
Can't use _search, object is needed instead of only its id